scalafix-core/shared/src/main/scala/scalafix/config/CustomMessage.scala

```diff
@@ -0,0 +1,47 @@
+package scalafix
+package config
+
+import metaconfig.{Conf, ConfError, ConfDecoder, Configured, Metaconfig}
+import org.langmeta._
+import scalafix.internal.config.MetaconfigPendingUpstream.XtensionConfScalafix
+
+import scala.language.implicitConversions
+
+class CustomMessage[T](
+    val value: T,
+    val message: Option[String],
+    val id: Option[String])
+
+object CustomMessage {
```

Comment:
Should we add a `apply(value: T) = new CustomMessage(value, None, None)` constructor?

Reply:
ibid. you only read from the conf.

Reply:
You're right, nevermind!